README.md

```diff
@@ -1,8 +1,15 @@
+## :warning: Deprecation warning
+In preparation for the KiCad v6 release this repository will be locked down on 2020-09.01. No new pull-requests will be accepted. Existing pull-requests can be worked on until 2020-10-01. Changes breaking v5.1 compatibility can be merged starting 2020-09-01.
+
+On 2020-10-01 this repository, including issues and pull requests, will be archived and transferred to gitlab.com. In order for the gitlab importer to function properly it is recommend you set your email-address on github to public and use the same address for you gitlab account. Otherwise the importer can't correlate the account information and the issues/comments on gitlab will be owned by `kicad-bot`.
```

Comment:
`recommended` and `your`

Also, doesn't the email address on GL have to be public, too?

Reply:
Unsure. I guess @sethhillbrand knows that?

Reply:
From the V6 timeline-thread:
> E-mail address on GH needs to be public and the same as one of the GL addresses to match users.

I also found the documentation for that importer:
https://docs.gitlab.com/ee/user/project/import/github.html#how-it-works

How about we add this link to the readme?

Reply:
Ah. Of course. It's a feature internal to GL and so doesn't care whether your email address there is public or not. Makes sense.

Adding the link is good idea.
Perhaps we should also mention that a GL account linked to GH via OAuth works, too.